go.mod

```diff
@@ -0,0 +1,34 @@
+module github.com/ONSdigital/dp-apipoc-server
+
+go 1.17
+
+require (
+	github.com/ONSdigital/dp-net/v2 v2.2.0-beta
+	github.com/ONSdigital/log.go/v2 v2.0.9
+	github.com/cznic/mathutil v0.0.0-20170313102836-1447ad269d64
+	github.com/gorilla/mux v1.8.0
+	github.com/jmoiron/jsonq v0.0.0-20150511023944-e874b168d07e
+	github.com/justinas/alice v1.2.0
+	github.com/kelseyhightower/envconfig v1.4.0
+	github.com/mattn/go-isatty v0.0.13 // indirect
+	github.com/rs/cors v1.8.0
+	github.com/smartystreets/goconvey v1.6.4
+	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c // indirect
+	golang.org/x/text v0.3.7 // indirect
+	gopkg.in/olivere/elastic.v3 v3.0.62
+	gopkg.in/tylerb/graceful.v1 v1.2.14
+)
+
+require (
```

Comment:
I don't know why the go tool is creating multiple `require` sections but I noticed I had some build issues with CI after upgrading go because of this. If you have similar issues, this is something you could look at but t could have just been a red herring.

Reply:
So go version 1.17.* splits dependencies into two sections - direct dependencies and indirect ones - if it becomes an issue on build then I will look to update this.